src/content/1.7/scale/_index.md

```diff
@@ -7,6 +7,34 @@ chapter: true
 
 ### Chapter 8
 
-# How to make PrestaShop scale
+# Scale
 
-Follow this guide to optimize your PrestaShop shop.
+Scaling your PrestaShop instalation is something we hope every merchant will need, because it means her business is growing.
```

Comment:
> her business is growing

Alternatively you can use a gender-neutral "their" -> "their business is growing"